src/cascadia/TerminalApp/ActionArgs.cpp

```diff
@@ -258,4 +259,28 @@ namespace winrt::TerminalApp::implementation
         return RS_(L"ResetTabNameCommandKey");
     }
 
+    winrt::hstring ToggleTabSwitcherArgs::GenerateName() const

```

Comment:
That's a good point, the anchor mode will pop up and not disappear bc there's no KeyUpEvent detected. I do agree on hiding anchor mode from the palette though, it doesn't make much sense to toggle it in that way.

I also found a related bug where toggling the non-anchor mode thru the palette doesn't work because the action that's dispatched  finishes before `_close()` is called inside of `_dispatchCommand()`, I'll have to add that to my TODOs.